README.md

```diff
@@ -29,7 +29,42 @@ On many systems, the following hugepages setup is sufficient:
 
 ### Option 1: Obtain Packages
 
-Debian packages for Gatekeeper aren't yet available, but will be soon.
+Debian packages for Gatekeeper aren't yet available, but will be soon at the
+[Releases](https://github.com/AltraMayor/gatekeeper/releases)
+page.
+
+#### Install
+
+Once the packages are downloaded, they can be installed with the commands below:
+
+    # dpkg -i libgkrte-*.deb \
+        gatekeeper-dpdk_19.02.0-0_amd64.deb \
+        gatekeeper-dpdk-dev_19.02.0_amd64.deb \
+        gatekeeper-dpdk-igb-uio-dkms_19.02.0_amd64.deb \
+        gatekeeper-dpdk-igb-uio-dkms_19.02.0_amd64.deb \
+        gatekeeper-bird_2.0.4-0_amd64.deb \
+        gatekeeper-1.0.0-0_amd64.deb
+
+#### Configuration
+
+Edit the `/etc/gatekeeper/envvars` file and insert the PCI addresses of the
+network adapters to be bound to DPDK. For example:
+
+    GATEKEEPER_INTERFACES="0000:00:07.0 0000:00:08.0"
```

Comment:
I didn't know we could use the PCI addresses here. But, for most people, it would be helpful to use the name of the interfaces. Mention the PCI address and the example above as an alternative form.

Reply:
Done.